$MirOS$

The first chunk must contain @ETC@ three times,
but /etc/nologin must not be changed (d'uh).

--- imap/src/imapd/Makefile.orig	Tue Jun 29 21:26:06 2004
+++ imap/src/imapd/Makefile	Sun Mar 19 18:33:50 2006
@@ -18,11 +18,11 @@
 # CPYRIGHT, included with this Distribution.
 
 
-ALERT=/etc/imapd.alert
+ALERT=@ETC@/imapd.alert
 USERALERT=.imapalert
 SHUTDOWN=/etc/nologin
-ANO=/etc/anonymous.newsgroups
-NNTP=/etc/imapd.nntp
+ANO=@ETC@/anonymous.newsgroups
+NNTP=@ETC@/imapd.nntp
 SHELL= /bin/sh
 
 
